napari/layers/surface/surface.py

```diff
@@ -209,6 +212,10 @@ def _get_extent(self):
             maxs = np.max(self.vertices, axis=0)
             mins = np.min(self.vertices, axis=0)
 
+        if self.vertex_values.ndim > 1:
+            mins = [0] * (self.vertex_values.ndim - 1) + list(mins)
+            maxs = list(self.vertex_values.shape[:-1]) + list(maxs)
```

Comment:
Can you add a comment about why this is necessary?

Reply:
comment added